parsers/julia.c

```diff
@@ -846,6 +886,29 @@ static void addTag (vString* ident, const char* type, const char* arg_list, int
     makeTagEntry(&tag);
 }
 
+static void addReferenceTag (vString* ident, int kind, int role, unsigned long line, MIOPos pos, vString* scope)
+{
+    if (kind == K_NONE)
+    {
+        return;
+    }
+    tagEntryInfo tag;
+    // HELP WANTED: for some reason initRefTagEntry doesn't work
+    // initRefTagEntry(&tag, vStringValue(ident), kind, role);
```

Comment:
Use initRefTagEntry. 
`--extras=+r` emits the tag initialized with this function.

Reply:
Done.